### DISCUSSION TITLE
Where userstyles are stored locally on the computer?

### DISCUSSION THREAD

**Discussion by TuTAH1** [stylus/stylus#2761] [Q&A] [answered]
I reinstalled browser and somewhy all userstyles are gone. Last userstyles backup was made a whole half year ago. But! Before reinstalling the browser, I made a backup of a whole browser folder (including User Data), so how can I get my userstyles from a backup?

**Comment by iChenLei** [comment, marked as answer]
goto [openstylus/stylus](https://github.com/openstyles/stylus)

**Reply by torrentails** [reply]
This doesn't actually answer the question.
